akka-actor/src/main/resources/reference.conf

```diff
@@ -350,6 +350,19 @@ akka {
         fallback = "fork-join-executor"
       }
 
+      affinity-pool-executor {
+        # The number of threads that this executor will maintain in its pool
+        num-threads = 8
```

Comment:
No, the min-max configuration structure is just to give upper and lower bounds.
See https://github.com/akka/akka/blob/master/akka-actor/src/main/resources/reference.conf#L357 and https://github.com/akka/akka/blob/master/akka-actor/src/main/resources/reference.conf#L365 and https://github.com/akka/akka/blob/master/akka-actor/src/main/resources/reference.conf#L362